### DISCUSSION TITLE
folder closed by default

### DISCUSSION THREAD

**Discussion by xu4wang** [directus/directus#12064] [Archive]
Is there a way to set the folder in the app closed by default when a user open the app?

<img width="965" alt="image" src="https://user-images.githubusercontent.com/311397/157607624-d49587ef-edd3-45cb-910f-8984a2b8b901.png">

For now, it's open by default, and user can see collection2,3,4 under the folder.

<img width="965" alt="image" src="https://user-images.githubusercontent.com/311397/157601922-1f747b0b-05c3-4344-8c76-a707312010e3.png">

👆 this is what I want. 

BR,Austin

**Comment by benhaynes** [comment]
Yeah, it's just hard to find... we can improve. Click this icon under **Settings > Data Model**

![Screen Shot 2022-03-10 at 9 01 41 AM](https://user-images.githubusercontent.com/522079/157677644-172f1996-6e1d-412e-a635-4d55bb02aa1a.png)

_Options include:_
* Start Collapsed
* Start Open
* Always Open

**Reply by xu4wang** [reply]
cool.  verified works. thanks Ben
